### ISSUE TITLE
Predefined search to find all TOTP entries

### ISSUE THREAD

**Issue by soredake** [keepassxreboot/keepassxc#9362]
## Summary
[TIP]:  # ( DO NOT include screenshots of your actual database! )
[NOTE]: # ( Provide a brief overview of what the new feature is all about )
It would be nice if keepassxc have dedicated OTP page like in google authentificator.

## Examples
[NOTE]: # ( Show us a picture or mock-up of your proposal )
![google auth](https://user-images.githubusercontent.com/5204968/234254315-71e6e094-70fa-4fb2-8de7-4ae413910407.jpg)

## Context
[NOTE]: # ( Why does this feature matter to you? What unique circumstances do you have? )

**Comment by droidmonkey** [comment]
Interesting idea

**Comment by varjolintu** [comment]
I like this one. Especially handy for those who have a separate 2FA database.

**Comment by LifeSteala** [comment]
+1 - A great idea.

**Comment by thepra** [comment]
I haven't figured out if KeePassXC does support more than 1 TOTP, the management UI for this feature exists only as menu options, which is insidious imo

**Comment by droidmonkey** [comment]
One TOTP per entry, unlimited entries

**Comment by varjolintu** [comment]
Something like this? :)
<img width="912" alt="Screenshot 2023-09-06 at 15 37 32" src="https://github.com/keepassxreboot/keepassxc/assets/24570482/7de7428d-d327-4c9c-ac51-10aaf9f7e252">

EDIT: It's just a first draft for the whole thing. Don't expect it to be ready soon.

**Comment by LifeSteala** [comment]
What would be cool is to "tag" all your 2FA entries. When you click on the tag it presents you the screen that @varjolintu has mocked up.

I have many entries across different folders for different purposes, and use tags to access them

**Comment by varjolintu** [comment]
@LifeSteala You can use the left sidebar to view all entries per a certain tag.

**Comment by droidmonkey** [comment]
I agree this could be easily implemented as a pre-defined search in the tags view.
